thunder/__init__.py

```diff
@@ -823,8 +785,6 @@ def fn_(*args, **kwargs) -> Any:
 
         cache_entry, inps, pro_to_epi = get_computation_and_inputs(*args, **kwargs)
 
-        check_storage_aliases(cache_entry, inps)
```

Comment:
if we do not have a test case to validate this, I'm not sure what this check is trying to do.